meshtastic/tests/test_node_runtime_seturl.py

```
@@ -0,0 +1,1498 @@
+            _make_channel(0, channel_pb2.Channel.Role.PRIMARY, "old"),
+            _make_channel(1, channel_pb2.Channel.Role.DISABLED),
+        ]
+
+        # Mock _raise_interface_error to raise an exception with the actual message
+        def raise_error(msg: str) -> NoReturn:
+            raise Exception(msg)
+
+        mock_local_node._raise_interface_error = MagicMock(side_effect=raise_error)
+
+        channel_set = apponly_pb2.ChannelSet()
+        for i in range(2):
```

Ruff (0.15.6)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)